### ISSUE TITLE
devtools::check() fails at x$is_alive()

### ISSUE THREAD

**Issue by aftonsteps** [ThinkR-open/golem#587]
Running `devtools::check()` fails this test:

```
Failure (test-golme-recommendedR:29:5): app lauches
------
x$is_alive() is not TRUE

`actual`: FALSE
`expected`: TRUE
```

I have tried with the version of `golem` on CRAN, the version on GitHub on the branch `master`, and the version on GitHub on the branch `dev`, all with the same result.

Feel free to move this to Discussions if this is a 'me' issue and not a 'golem' issue, though!

**Comment by ColinFay** [comment]
Hey, 

Yeah finding a way to do that has been a mess. 

If ever you're interested in the details, it's explained here why it's complicated: 
+ https://github.com/ThinkR-open/golem/blob/dev/R/test_helpers.R#L71

Can you try the dev version of golem (`remotes::install_github("thinkr-open/golem", ref = "dev", force = TRUE)`) and let me know if this works? 

Thanks, 
Colin

**Comment by aftonsteps** [comment]
Ah, I see! Thank you. I tried each way in the comments and it seems to work! Here are the results:

(1) Running the test interactively (i.e cmd A + cmd Enter): ✅
(2) Running the test inside the console, with devtools::test(): ✅
(3) Running the test inside the console, with devtools::check(): ✅
(4) Clicking on the "Run Tests" button on test file File: ✅
(5) Clicking on the "Test" button in RStudio Build Pane: ✅
(6) Clicking on the "Check" button in RStudio Build Pane: ✅

I did have to not use `golem::use_utils_ui()` in order for the tests to pass for (1), (3), and (6), but I think that's a separate issue. When I tried (1), I was running `03_deploy.R` which includes `devtools::check()`, so it seems to be an issue with `check()` and not `test()`. Here's what happens if I do include the `golem_utils_ui.R` file, and then run the checks as in (1), (3), and (6):

```
  Running examples in ‘golemtest-Ex.R’ failed
  The error most likely occurred in:
  
  > base::assign(".ptime", proc.time(), pos = "CheckExEnv")
  > ### Name: list_to_p
  > ### Title: Turn an R list into corresponding HTML paragraph tags
  > ### Aliases: list_to_p
  > 
  > ### ** Examples
  > 
  > list_to_p(c("This is the first paragraph", "this is the second paragraph"))
  Error in list_to_p(c("This is the first paragraph", "this is the second paragraph")) : 
    could not find function "list_to_p"
  Execution halted
```
